### PR TITLE
Update README for Cocoapods

### PR DESCRIPTION
**Pull request by ziyiz-amzn**
### Description of changes:
Update README for Cocoapods

#### Manual test cases (add more as needed):

- [ ] Join meeting without CallKit
- [ ] Join meeting with CallKit as Incoming
- [ ] Join meeting with CallKit as Outgoing
- [ ] Leave meeting
- [ ] Rejoin meeting
- [ ] See metrics
- [ ] See attendee presence
- [ ] Send audio
- [ ] Receive audio
- [ ] Mute self
- [ ] Unmute self
- [ ] See local mute indicator when muted
- [ ] See remote mute indicator when other is muted
- [ ] Enable and disable local video
- [ ] Enable and disable remote video from remote side
- [ ] Send local video
- [ ] Pause and resume remote video
- [ ] Switch local camera
- [ ] Receive remote screen share

#### Integration validation (required before release)

- [ ] Unit tests pass
- [ ] Build SDK against simulator with release options
- [ ] Build SDK against device with release options
- [ ] Build SDK against simulator with debug options
- [ ] Build SDK against device with debug options
- [ ] Test Demo against simulator with SDK (debug build) in workspace
- [ ] Test Demo against device with SDK (debug build) in workspace
- [ ] Test DemoObjC against simulator with SDK (debug build) in workspace
- [ ] Test DemoObjC against device with SDK (debug build) in workspace
- [ ] Test Demo against simulator with SDK.framework (release build)
- [ ] Test Demo against device with SDK.framework (release build)
- [ ] Test DemoObjC against simulator with SDK.framework (release build)
- [ ] Test DemoObjC against device with SDK.framework (release build)

### Screenshots, if available:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by georgezy-amzn — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -39,8 +39,16 @@ And review the following guides:
...
 ## Include Amazon Chime SDK in Your Project
+You can download and integrate the Amazon Chime iOS SDKs to your project from both CocoaPods and binaries from Github release.
 
-To include the SDK binaries in your own project, follow these steps.
+### From CocoaPods
+To obtain Amazon Chime iOS SDKs from CocoaPods, add 
+```
+pod 'AmazonChimeSDK-Bitcode'
+```
+to `Podfile`. The no bitcode version `AmazonChimeSDK-No-Bitcode` is also available in [Cocoapods](https://cocoapods.org/).
```
Comment (linsang21) [inline README.md:49]: Is it necessary to link cocopods here?

**Review by linsang21 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -39,8 +39,16 @@ And review the following guides:
 * [Configuring Remote Video Subscription](guides/configuring_remote_video_subscriptions.md)
 
 ## Include Amazon Chime SDK in Your Project
+You can download and integrate the Amazon Chime iOS SDKs to your project from both CocoaPods and binaries from Github release.
```
Comment (zhinang-amazon) [inline README.md:42]: You can integrate Amazon Chime SDK in your project from either CocoaPods or binaries through Github release.

File: README.md [hunk 3]
```diff
@@ -39,8 +39,16 @@ And review the following guides:
...
 
-To include the SDK binaries in your own project, follow these steps.
+### From CocoaPods
```
Comment (zhinang-amazon) [inline README.md:44]: If builder is using cocoapods, do they still need the second step of updating project file?
If not, "From CocoaPods" and "From Github Release Binaries" should be a level higher than "1 Download Binaries" and "2. Update Project File"

File: README.md [hunk 4]
```diff
@@ -39,8 +39,16 @@ And review the following guides:
...
+To obtain Amazon Chime iOS SDKs from CocoaPods, add
```
Comment (zhinang-amazon) [inline README.md:45]: Amazon Chime iOS SDKs -> Amazon Chime SDK

Comment (zhinang-amazon) [reply README.md:45]: This section should also be a "If else" on whether builder needs bitcode or not.
Can you reference [this](https://github.com/aws-amplify/aws-sdk-ios#cocoapods) to provide a detailed guide?

**Review by zhinang-amazon — APPROVED** [review]

**Review by georgezy-amzn — APPROVED** [review]

**Pull request merged**